modules/serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-scaling-up-workflows-with-gitops-profile_{context}"]
+= Scaling up workflows with the gitops profile
+
+Workflows with the `gitops` profile that were previously scaled down must be scaled back up to continue operation.
+
+.Procedure
+
+. Modify the `my-workflow.yaml` CR and scale up each workflow to `one` as shown in the following example:
++
+[source,yaml]
+----
+spec:
+  podTemplate:
+    replicas: 1
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks 

Consider applying this change to all of the other similar places in this PR

> For snippets or sections of a file, use an ellipsis (…​) to show that the file continues before or after the quoted block. For YAML, use ('#…​') instead.